avocado/plugins/assets.py

```diff
@@ -208,6 +208,9 @@ def fetch_assets(test_file, klass=None, method=None):
             call['cache_dirs'] = cache_dirs
             call['expire'] = expire
             asset_obj = Asset(**call)
+            if logger is not None:
+                logger.info('Fetching asset from %s:%s.%s',
+                            test_file, klass, method)
```

Comment:
I always try to keep the output messages in the caller function, in this case, `pre_tests`. With this practice, I know I need to handle the output message when I'm creating a new class that makes use of other functions, and I don't need to worry if those functions will throw something in the output. As you added a parameter to control this behavior, I will leave it up to you to decide if it is possible to move the log messages to the caller method or control it with a parameter.

Reply:
@willianrampazzo Yeah, this is not ideal, but I don't want users to see "fetching asset..." lines when there are no assets to be fetched - this is of the uttermost priority IMO.

The problem is that a different solution would require a finer grained set of asset fetching utilities, such as one listing the assets available, and another showing the asset fetching result.  So, I avoided making a deep surgery on this code right now, and tried to solve the biggest issue as simple as possible.  Like I've stated here:

https://trello.com/c/lumR8u8Y/1526-rfc-nrunner-extended-metadata

I believe it's the loader/resolver responsibility to collect that kind of information and make it available in the test suite for any plugin.

Let me know if you think a different solution can be done in time for the next release (hopefully by Tomorrow) or if you think this is OK for now, and an improved version could be added later.

Reply:
No problem on my side to merge this code now. I agree that to comply with what I mentioned, we would need a bigger code refactor. As I have a todo to implement a granular `assets fetch` command, it can be refactored later.

Reply:
Got it, thanks!